### DISCUSSION TITLE
horsesConverter Model

### DISCUSSION THREAD

**Discussion by Dewu-Yang** [loganoz/horses3d#170] [Q&A] [answered]
Dear all,
   How should one proceed to import a new grid during restart and continue the calculations on the new grid?How should I use the horsesConvertor module? Thanks！
Best,
Dewu

**Comment by hmarbona** [comment, marked as answer]
Hi Dewu-Yang,
I realized there is a bug on the version pushed to the master, its printing NaN, I will fix that before the end of week and explain in in more detail

Himpu

**Reply by Dewu-Yang** [reply]
Dear Prof.hmarbona,

OK,thank you for your reply!

Best,
Dewu

**Reply by hmarbona** [reply]
Hi Dewu,
I wont be able to push it to master due to different problem, maybe until early of september. If you still need it, could you share your email so I can send you a file to be change?

Cheers,
Himpu
